### DISCUSSION TITLE
While converting the excel wide format MRF to the JSON format how to handle properties with empty values?

### DISCUSSION THREAD

**Discussion by MS2606** [CMSgov/hospital-price-transparency#98] [Q&A] [answered]
example - for the code 470 there is a negotiated rate with payer-1 but has no negotiated rate with payer-2. while encoding code 470 in JSON format if payer-1 information is dropped (since there is no rate & it is an empty cell) it generates "property missing" error in the validator. similar is noticed for the minimum, maximum and discounted cash. Even though all 3 properties are present in the MRF for the other codes, validator generates "property missing" error based on the evaluation of the very first code in the MRF

**Comment by RidenourK** [comment, marked as answer]
@MS2606. Thank you for your question.
 
CMS strongly recommends hospitals start by downloading one of the template layouts or schema to create the machine-readable file, as opposed to converting an existing file. Please then review the technical instructions in the “[Documentation](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation)” section, followed by the “[Examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples)” available on this the CMS Hospital Price Transparency - Data Dictionary GitHub repository for more information about how to encode data in one of the CMS templates.
